CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+## Contributing Issues
+
+### Before Submitting an Issue
+First, if your issue is a question or you need help with setting up / configuring Streisand, then please ask on the [streisand-discussions](https://github.com/jlund/streisand-discussions/) repo.
+
+If a bug or technical feature request, please do a search in [open issues](https://github.com/jlund/streisand/issues) to see if the issue or feature request has already been filed.
+
+If you find your issue already exists, make relevant comments and add your [reaction](https://github.com/blog/2119-add-reactions-to-pull-requests-issues-and-comments). Use a reaction in place of a "+1" comment.

```

Comment:
Is there a way to define "relevant comments" more directly? It would be nice to make that more concrete (e.g. new information, additional repro steps are "relevant", the me too stuff isn't).